src/lib/core/components/visualizations/implementations/BoxplotVisualization.tsx

```diff
@@ -312,6 +315,73 @@ function BoxplotViz(props: VisualizationProps) {
       ? data.value?.completeCasesAllVars
       : data.value?.completeCasesAxesVars;
 
+  const dependentAxisRange: NumberOrDateRange | undefined = useMemo(() => {
+    if (isFaceted(data?.value)) {
+      // may not need to check yAxisVariable?.type but just in case
+      return data?.value?.facets != null &&
+        (yAxisVariable?.type === 'number' || yAxisVariable?.type === 'integer')
+        ? {
+            min:
+              (min([
+                0,
+                min(
+                  data.value.facets.flatMap((facet) =>
+                    facet.data.series
+                      .flatMap((o) => o.outliers as number[][])
+                      .flat()
+                  )
+                ),
+                min(
+                  data.value.facets.flatMap((facet) =>
+                    facet.data.series.flatMap((o) => o.lowerfence as number[])
+                  )
+                ),
+              ]) as number) * 1.05,
```

Comment:
I applied 5% increase for both min and max, just in case, i.e., multiplied by 1.05 regardless of +/-. So for example, if min and max is assumed to be [-1, 1], then actual plot range will be [-1.05, 1.05]: -1.05 is smaller than -1 😃